prov/shm/src/smr_progress.c

```diff
@@ -288,8 +293,9 @@ static int smr_progress_iov(struct smr_cmd *cmd, struct iovec *iov,
 }
 
 static int smr_mmap_peer_copy(struct smr_ep *ep, struct smr_cmd *cmd,
-				 struct iovec *iov, size_t iov_count,
-				 size_t *total_len)
+			      enum fi_hmem_iface iface, uint64_t device,
+			      struct iovec *iov, size_t iov_count,
+			      size_t *total_len)
```

Comment:
Indentation: Pls start right after the column of '(' on each wrapped line.

Reply:
Like the the way you did on L329

Reply:
This seems to be a rendering error on github, it is turning one of the tabs into a space in the preview for some reason. If you view the file at this revision you can see it is aligned properly. The preview below is also wrong in a different way, but if you click through to the file you will see it is correct.
https://github.com/ofiwg/libfabric/blob/ca59a1c5767e5ab486481fa43aad3aad768167aa/prov/shm/src/smr_progress.c#L295-L298

This goes for the other comment you left as well.

Reply:
huh, the embedded preview is correct, but it isn't correct when I preview the comment before I send it. Github is doing some really weird stuff with its whitespace rendering...